### DISCUSSION TITLE
Postprocessing citations/references generated generated from CSL styles

### DISCUSSION THREAD

**Discussion by J-Moravec** [quarto-dev/quarto-cli#4342] [Q&A]
I would like to highlight (bold) my name in bibliography, I got it with bibtex `.bst` style presentation, but I would like to have that on a website in a HTML form.

For past few hours, I have been learning CSL and trying to figure out how to do that only to finally learn that it is impossible, see: https://discourse.citationstyles.org/t/underline-specific-authors/1752/2

According to the discussion, the only way how this can be done is to post-process the markdown/html-generated output and add tags.

I would guess that this would be a commonly-requested feature for personal websites. Who wants to typeset all the stuff manually, right? (multiple bibliographies would also be cool, division by year makes them quite a bit more cleaner)

**Comment by mcanouil** [comment]
You could do that with post-render script, see <https://quarto.org/docs/projects/scripts.html#pre-and-post-render>.
To note, this is a Quarto project feature.
